src/uwtools/utils/file.py

```diff
@@ -39,7 +40,9 @@ class _FORMAT:
     ini: str = _ini
     jinja2: str = _jinja2
     nml: str = _nml
+    rocoto: str = _xml
     sh: str = _ini
+    xml: str = _xml
```

Comment:
I may have missed it, but I don't think `FORMAT.xml` was ever used and `FORMAT.rocoto` was used arbitrarily, so may not be needed here, either.